Realm/RLMRealm.h

```diff
@@ -266,10 +274,14 @@ typedef void(^RLMNotificationBlock)(NSString *notification, RLMRealm *realm);
 
  If set to YES (the default), changes made on other threads will be reflected
  in this Realm on the next cycle of the run loop after the changes are
- committed.  If set to NO, you must manually call -refresh on the Realm to
+ committed.  If set to NO, you must manually call `-refresh` on the Realm to
  update it to get the latest version.
 
- Even with this enabled, you can still call -refresh at any time to update the
+ Note that on background threads, the run loop is not run by default and you will 
```

Comment:
Apple’s docs are inconsistent (as usual), but it looks like at some point they changed the language from “Every thread has a run loop associated” to “automatically created as needed”. I’ll update this in the Swift docs as well.